### PR TITLE
Add support for Polar Heart Rate Sensor (PT-187250791)

### PR DESCRIPTION
**Pull request by lublagg**

### CODE REVIEW

File: src/models/sensor-polar-manager.ts [hunk 1]
```diff
@@ -0,0 +1,243 @@
...
+    this.supportsHeartbeat = true;
+  }
+
+  async connectToDevice(wirelessDevice: BluetoothDevice): Promise<boolean> {
+    this.device = wirelessDevice;
+
+    // Connect to device
+    this.server = await this.device?.gatt?.connect();
+
+    // Resend the sensorconfig so the UI udpates after the connection
```
Comment (dougmartin) [inline src/models/sensor-polar-manager.ts:83]: udpates => updates

File: src/models/sensor-polar-manager.ts [hunk 2]
```diff
@@ -0,0 +1,243 @@
...
+    }
+    this.device?.gatt?.disconnect();
+    // Resend the sensorconfig so the UI udpates after the disconnection
+    this.sendSensorConfig(true);
+  }
+
+  handleHeartRateMeasurement(heartRateMeasurement: BluetoothRemoteGATTCharacteristic) {
+    heartRateMeasurement.addEventListener('characteristicvaluechanged', (event: any) => {
+      const heartRateMeasurement = this.parseHeartRate(event.target?.value).heartRate;
+      this.printByteArray(event.target?.value);
```
Comment (dougmartin) [inline src/models/sensor-polar-manager.ts:111]: Do you still need this for debugging?

**Review by dougmartin — APPROVED** [review]
Looks good :+1:

File: src/models/sensor-polar-manager.ts (hunk 2 above)
Comment (lublagg) [reply src/models/sensor-polar-manager.ts:111]: Yes, it's helpful to keep there for debugging purposes.

File: src/models/sensor-polar-manager.ts (hunk 1 above)
Comment (lublagg) [reply src/models/sensor-polar-manager.ts]: whoops! good catch.

**Pull request merged**